compiler/tests/success/subarrays/x86-64/unaligned_cast.jazz

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
This file has been added to the test-suite of the compiler (the tool that produces assembly). Yet, it is rejected by said compiler. Is that intended?

Reply:
That was a bug.